modules/oadp-installing-dpa.adoc

```diff
@@ -54,40 +54,42 @@ spec:
       defaultPlugins:
         - openshift <1>
         - aws
+      resourceTimeout: 10m <2>
     restic:
-      enable: true <2>
+      enable: true <3>
       podConfig:
-        nodeSelector: <node selector> <3>
+        nodeSelector: <node selector> <4>
   backupLocations:
     - name: default
       velero:
         provider: {provider}
         default: true
         objectStorage:
-          bucket: <bucket_name> <4>
-          prefix: <prefix> <5>
+          bucket: <bucket_name> <5>
+          prefix: <prefix> <6>
         config:
           region: <region>
           profile: "default"
         credential:
           key: cloud
-          name: {credentials} <6>
-  snapshotLocations: <7>
+          name: {credentials} <7>
+  snapshotLocations: <8>
     - name: default
       velero:
         provider: {provider}
         config:
-          region: <region> <8>
+          region: <region> <9>
           profile: "default"
 ----
 <1> The `openshift` plugin is mandatory.
-<2> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that each worker node has `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
-<3> Specify the node selector to be supplied to Restic podSpec.
-<4> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
-<5> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
-<6> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
-<7> You do not need to specify a snapshot location if you use CSI snapshots or Restic to back up PVs.
-<8> The snapshot location must be in the same region as the PVs.
+<2> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
```

Comment:
It seems that you are still waiting on some clarification on this but here's a rewording suggestion:

```suggestion
<2> Specify the time, in minutes, to wait for Velero resources, before timeout occurs. The default is 10m.
```

Reply:
@sjhala-ccs this was already approved by in a different PR that the QE person is unaware of. Thank you for the very helpful review. In this particular instance, I'm going to stay with the original text.